ui/src/app/views/workspace/component.ts

```diff
@@ -129,12 +135,27 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
       this.researchPurposeArray.slice(
         this.leftResearchPurposes.length,
         this.researchPurposeArray.length);
-    this.showTip = true;
+    this.showTip = false;
+    this.pageId = 'workspace';
```

Comment:
nit: I think a static constant would make more sense for these IDs, e.g. https://github.com/all-of-us/workbench/blob/cefa291bc43f2a3f79f83ac5ff6ba42337008c5c/ui/src/app/services/profile-storage.service.ts#L16